packages/venia-concept/src/components/ProductFullDetail/productFullDetail.css

```diff
@@ -101,15 +101,16 @@
 }
 
 .cartActions {
-    composes: section;
-    align-items: center;
-    display: flex;
-    grid-area: cart;
-    justify-content: center;
+    display: none;
```

Comment:
Am I reading this correctly that this will be hidden at mobile viewport sizes?

Reply:
@supernova-at I believe that desktop layout doesn't suit to some kinds of sticky things like button